packages/components/src/lozenge/docs/Lozenge.stories.mdx

```diff
@@ -19,7 +19,6 @@ import { Text } from "@components/typography";
     githubPath="/packages/components/src/lozenge/src"
 />
 
-
```

Comment:
Should we add a mention that changing the background or color of the Lozenge can lead to accessibility failures? Something along the lines of 
> Lozenge's variants follow the contrast guidelines. Changing the background or text color might cause contrast check failures.
(Not sure `contrast check failures` is the right way to put it)

Reply:
errors? issues? noncompliance?

Reply:
actually we don't want people to do it but we understand that we might need to do it and at least it won't fail. I will add it to the accessibility section of the lozenge doc, but we don't document it for other components like button / tag who also support changing the backgroundColor and color.